packages/manfred2md/src/engine/md-parts/experience-section/experience-section.ejs

```diff
@@ -4,18 +4,32 @@
 <%_ if (experienceCollection && experienceCollection !== 'undefined') { -%>
   <%_ for (const job of experienceCollection) { -%>
   - #### <%- job.name %>
+    <%_ if (job.type && job.type !== 'undefined' ) { -%>
     - **Tipo de Organización:** <%- job.type %>
+    <%_ } -%>
+    <%_ if (job.description && job.description !== 'undefined' ) { -%>
     - **Descripción:** <%- job.description %>
+    <%_ } -%>
     - **Roles dentro de la empresa:**
     <%_ if (job.roles && job.roles !== 'undefined' ) { -%>
       <%_ for (const role of job.roles) { -%>
+      <%_ if (role.name && role.name !== 'undefined' ) { -%>
       - **<%= role.name %>**
+      <%_ } -%>
+        <%_ if (role.startDate && role.startDate !== 'undefined' && role.finishDate !== 'undefined' ) { -%>
```

Comment:
startDate is a required field, leave it as it was.
- **Duración:** <%= role.startDate %> - <%= role.finishDate ?? 'Actualidad' %>